src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -268,47 +269,27 @@ public static double sensitivityAtConstantDepth(final int depth, final Histogram
             final int altDepth = bd.sample();
 
             int sumOfQualities;
-            if (altDepth < LARGE_NUMBER_OF_DRAWS) {
-                // If the number of alt reads is "small" we draw from the actual base quality distribution.
-                //sumOfQualities = IntStream.range(0, altDepth).map(n -> qualityRW.draw()).sum();
-                sumOfQualities = 0;
-                for(int i = 0;i < altDepth;i++) {
-                    if(rg.nextDouble() > overlapProbability) {
-                        sumOfQualities += Math.min(qualityRW.draw(), pcrErrorRate);
-                    } else {
-                        // Simulate overlapping reads by sampling from the quality distribution twice.
-                        // Since PCR errors affect overlapping reads, the total contribution to the sum
-                        // of qualities cannot exceed the PCR error rate.
-                        sumOfQualities += Math.min(qualityRW.draw() + qualityRW.draw(), pcrErrorRate);
-                    }
+            // If the number of alt reads is "small" we draw from the actual base quality distribution.
```

Comment:
Since it looks like you're getting rid of the Gaussian approximation at larger depth it seems that this comment is now moot.